packages/api/cms-api/src/dam/files/files.resolver.ts

```diff
@@ -103,6 +105,23 @@ export function createFilesResolver({ File, Scope: PassedScope }: { File: Type<F
             return this.filesService.moveBatch(files, targetFolder);
         }
 
+        @Mutation(() => CopyFilesResponse)
+        @SkipBuild()
+        async copyFilesToScope(
+            @GetCurrentUser() user: CurrentUserInterface,
+            @Args("fileIds", { type: () => [ID] }) fileIds: string[],
+            @Args("targetScope", { type: () => Scope }) targetScope: typeof Scope,
+            @Args("targetFolderId", {
```

Comment:
Wouldn't a better name be `existingInboxFolder`? I think you could shorten the description then. And the `Id` suffix can be omitted IMO, as we're using the `ID` scalar anyway.

Reply:
> Yes true. But I would still like to keep the Id suffix.

Maybe we should come up with a general recommendation in the future. Currently some add the suffix, others don't

Reply:
Yes. I consider api-generator as best-practice and recommondation. It creates no Id suffix ATM, see https://github.com/vivid-planet/comet/blob/next/demo/api/schema.gql#L1031

If you don't like that, let's discuss it there.

For existing apis (like dam) we should
- try to stay consistent within that feature
- maybe change to the new recommendation

Reply:
@thomasdax98 this should be consistent within the DAM feature, am I right?

Reply:
@johnnyomair @nsams 

Currently, I use the Id suffix everywhere in the files resolver.

- `folderId` in `damIsFilenameOccupied()`
- `targetFolderId` in `moveDamFiles()`
- `fileIds` in `moveDamFiles()`

So it's consistent within the DAM. 

I'm also fine with the new recommendation. I could migrate in a follow-up PR